app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  # Callback
+  after_save :updates_comments_counter
+
+  # Methods
+  def updates_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
[OPTIONAL]
- [ ] It will be nice to make the `updates_comments_counter` method private since we only call it inside the class.
```ruby
  private

  def updates_comments_counter
    post.update(comments_counter: post.comments.count)
  end
```

Reply:
- Method made private.